README.md

```diff
@@ -1098,6 +1098,21 @@ editor.on('deleteAllRows', deletedValues => {
 });
 ```
 
+#### Schema loader events
+
+When a schema is loaded through a request, the schemaLoaded event is triggered.
```

Comment:
Does this apply to any schema loaded or just the initial one?
The log message is in plural btw.